packages/runtime/id-compressor/src/test/snapshots/summary.spec.ts

```diff
@@ -0,0 +1,156 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/* eslint-disable import/no-nodejs-modules */
+
+import { strict as assert } from "assert";
+import { existsSync, rmSync, mkdirSync, writeFileSync, readFileSync } from "fs";
+import path from "path";
+import { SessionId, createIdCompressor } from "../..";
+
+const regenerateSnapshots = process.argv.includes("--snapshot");
```

Comment:
should there be something in the package.json for this?

Reply:
also, probably fine but just checking argv without knowing the command is a bit scary

Reply:
Added a script to the package.json for this. As for blindly checking argv, I just followed what we do in tree's directory but am open if you want to do it a diff way.